packages/react-component-library/src/enhancers/withFormik.tsx

```diff
@@ -16,7 +17,7 @@ export const withFormik = (FormComponent: React.FC<any>) => ({
   className,
   ...props
 }: FormikProps) => {
-  const hasError = touched[field.name] && errors[field.name]
+  const hasError = !!(touched[field.name] && errors[field.name])
```

Comment:
Maybe not one for here but one comment I got from our designer was he didn't want to see errors until the first submit, but this is outside the scope of this